src/Components/Atoms/Dropdown.jsx

```diff
@@ -0,0 +1,5 @@
+const Dropdown = (props) => {
```

Comment:
[destructuring](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)
을 적용하면 코드를 더 깔끔하게 작성할 수 있을 것 같네요. 
컴포넌트 props는 주석으로 타입이랑 간단한 설명 적어주시면 협업에 용이한 코드가 될 것 같네요 
추후에는 ts도입과 interface를 공부해보세요
```suggestion
const Dropdown = ({isOpen, children) => {
```